pkgs/development/libraries/mesa-darwin/default.nix

```diff
@@ -52,6 +52,11 @@ let
 
     configurePhase = ":";
 
+    configureFlags = [
+      # NOTE: In Mesa >=18.1.2 - enabled by default: patents on floating point textures expired.
+      (lib.enableFeature enableTextureFloats "texture-float")
+    ];
+
```

Comment:
Doesn't this block of changes actually disable the feature?

Reply:
Well, it should be no change from default on this mesa version, so I certainly don't feel strongly about it.